api/src/main/java/org/pmiops/workbench/compliance/ComplianceServiceImpl.java

```diff
@@ -25,12 +27,17 @@
   private CloudStorageService cloudStorageService;
   private Provider<WorkbenchConfig> configProvider;
 
+  private MoodleConfig moodleConfig;
+
 
   @Autowired
   public ComplianceServiceImpl(CloudStorageService cloudStorageService,
-      Provider<WorkbenchConfig> configProvider) {
+      Provider<WorkbenchConfig> configProvider,
+      Provider<MoodleConfig> moodleConfigProvider) {
```

Comment:
In my suggested approach, this would be `Provider<MoodleApi>` instead. The `moodleApi` method you defined will be called automatically by spring and injected.